ballerina/tests/resources/copybook-json/copybook-9.json

```diff
@@ -0,0 +1,9 @@
+{
+    "Employee": {
+        "EmployeeId": 2.345234,
+        "ContractYears": 323.33,
+        "Salary": 0000005.200,
+        "Bonus": 003412.000000,
```

Comment:
Are these values correct? In JSON we don't need the leading zeros, right?

Reply:
Changed!